package.json

```diff
@@ -15,38 +15,42 @@
   "scripts": {
     "start": "gulp watch",
     "build": "gulp",
-    "prepublish": "npm run build"
+    "prepack": "npm run build"
   },
   "peerDependencies": {
     "@frctl/fractal": ">= 1.1.0-alpha.0 < 2"
   },
   "devDependencies": {
+    "@frctl/fractal": "^1.2.1",
     "babel-preset-es2015": "^6.5.0",
     "babelify": "^7.2.0",
     "browserify": "^13.0.0",
-    "del": "^2.2.0",
-    "gulp": "^3.9.1",
-    "gulp-autoprefixer": "^3.1.0",
+    "del": "^5.1.0",
+    "gulp": "^4.0.2",
+    "gulp-autoprefixer": "^7.0.1",
     "gulp-sass": "^4.0.0",
     "gulp-sass-glob": "^1.0.0",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-stylelint": "^2.0.0",
-    "gulp-uglify": "^1.5.3",
+    "gulp-sourcemaps": "^2.6.5",
+    "gulp-stylelint": "^10.0.0",
+    "gulp-uglify": "^3.0.2",
     "highlight.js": "^9.1.0",
     "jquery": "^2.2.1",
     "jquery-pjax": "git+https://github.com/defunkt/jquery-pjax.git",
     "jquery-resizable-dom": "^0.16.0",
     "normalize.css": "^4.1.1",
     "sass-mq": "^3.2.6",
     "select2": "4.0.3",
-    "stylelint-config-standard": "^7.0.0",
-    "vinyl-buffer": "^1.0.0",
-    "vinyl-source-stream": "^1.1.0",
+    "stylelint": "^11.1.1",
+    "stylelint-config-standard": "^19.0.0",
+    "vinyl-buffer": "^1.0.1",
+    "vinyl-source-stream": "^2.0.0",
     "watchify": "^3.7.0"
   },
   "dependencies": {
-    "@frctl/fractal": "^1.1.0-alpha.0",
     "js-beautify": "^1.6.2",
     "lodash": "^4.12.0"
-  }
+  },
+  "browserslist": [
+    "last 5 versions"
```

Comment:
As discussed on Slack, let's remove this and use Browserslist default, which is closer to Browsers we usually supports.

Reply:
removed it